src/cljs/athens/components/website_embeds.cljs

```diff
@@ -0,0 +1,30 @@
+(ns athens.components.website-embeds)
+
+
+(def component-youtube-embed
+  {:match  #"\[\[youtube\]\]\:.*"
+   :render (fn [content uid]
+             [:span {:id (str content uid)}
```

Comment:
May want to create a generic span around all components that calls `(.. e stopPropagation)` in order to not drop into the textarea on click. Maybe `components-utils.default-component`